hackclub.com.yaml

```diff
@@ -1329,6 +1332,9 @@ psn:
   type: CNAME
   value: psnhackclub.netlify.app.
```

Comment:
they're linked to netlify, the new ones are for vercel

Reply:
these aren't for psn, it's for puyallup and phs

Reply:
ah sorry, my bad!

Reply:
It's alright! do you have any idea for my stuff though?